thirdparty/github.go

```diff
@@ -888,24 +888,28 @@ func GetGithubPullRequestCommits(ctx context.Context, token, owner, repo string,
 }
 
 // GetGithubPullRequestReviews retrieves a list of reviews for the given PR.
-func GetGithubPullRequestReviews(ctx context.Context, token, owner, repo string, prNumber int) ([]*github.PullRequestReview, error) {
+func GetGithubPullRequestReviews(ctx context.Context, token, owner, repo string, prNumber int, reviewPage int) ([]*github.PullRequestReview, int, error) {
 	httpClient := getGithubClientRetryWith404s(token, "GetGithubPullRequestCommits")
 	defer utility.PutHTTPClient(httpClient)
 
 	client := github.NewClient(httpClient)
 
 	opts := &github.ListOptions{
-		PerPage: 100,
+		Page: reviewPage,
 	}
-	reviews, _, err := client.PullRequests.ListReviews(ctx, owner, repo, prNumber, opts)
-	if err != nil {
-		return nil, err
-	}
-	if len(reviews) == 0 {
-		return nil, errors.New("no PR reviews received from GitHub")
+	reviews, resp, err := client.PullRequests.ListReviews(ctx, owner, repo, prNumber, opts)
+	if resp != nil {
+		defer resp.Body.Close()
+		if err != nil {
+			return nil, 0, parseGithubErrorResponse(resp)
+		}
+	} else {
+		errMsg := fmt.Sprintf("nil response from query for PR reviews in '%s/%s' prNumber %d : %v", owner, repo, prNumber, err)
+		grip.Error(errMsg)
```

Comment:
I imagine we would log this error anyways at a higher level. Do we need to log it here?

Reply:
removed